01-js-week-1/functions-worksheet.md

```diff
@@ -0,0 +1,400 @@
+# JavaScript Worksheet: Functions
+For Part 1, read the code in each section, then write the equivalent JavaScript code for the Ruby that is given. Start by writing it out in a text editor or on a piece of paper. Then try running it and tweak your code until it successfully runs with expected output.
+For Part 2, follow the directions for each problem.
+
+**Note:** For each function you create, you should **also write the code to call** the function.
+
+## Part 1
+<!--BEGIN CHALLENGE-->
+
+### !challenge
+
+* type: paragraph
+* id: f5499bb9-86ab-40cc-86b8-320306e2ebfc
+* title: Convert to JavaScript
+<!--Other optional fields (checkpoints only) -->
+<!--`points: 1`: the number of points for scoring as a checkpoint-->
+<!--`topics: python, pandas`: the topics for analyzing points-->
+
+##### !question
+
+Convert the following code into JavaScript, and then call the function:
+```ruby
+def hello
+  puts 'hello!'
+end
+```
+
+##### !end-question
+
+##### !placeholder
+
+your code here...
+
+##### !end-placeholder
+
+<!--optional-->
+##### !hint
+
+##### !end-hint
+
+<!--optional, checkpoints only-->
+##### !rubric
+
+##### !end-rubric
+
+<!--optional-->
+##### !explanation
+
+```javascript
+const hello = function() { 
+  console.log('hello!');
+}
+
+hello()
+```
+
+##### !end-explanation
+
+### !end-challenge
+
+<!--END CHALLENGE-->
+<!--BEGIN CHALLENGE-->
+
+### !challenge
+
+* type: paragraph
+* id: aeea65c1-cb0d-4126-b14d-7f01ae183556
+* title: Convert to JavaScript
+<!--Other optional fields (checkpoints only) -->
+<!--`points: 1`: the number of points for scoring as a checkpoint-->
+<!--`topics: python, pandas`: the topics for analyzing points-->
+
+##### !question
+
+Convert the following code into JavaScript, and then call the function:
+```ruby
+def say_num(number)
+  puts 'Your number is #{number}.'
+end
+```
+
+##### !end-question
+
+##### !placeholder
+
+your code here...
+
+##### !end-placeholder
+
+<!--optional-->
+##### !hint
+
+##### !end-hint
+
+<!--optional, checkpoints only-->
+##### !rubric
+
+##### !end-rubric
+
+<!--optional-->
+##### !explanation
+
+```javascript
+const sayNum = function (number) {
```

Comment:
Not sure why, but learn seems to strip the syntax highlighting on this explanation.

Reply:
ok I'll look into it and see if I made a syntax error

Reply:
This looks fixed from my end. Closing out the PR and linking!